src/app/account/account.cpp

```diff
@@ -0,0 +1,63 @@
+#include "account.hpp"
+
+Account::Account(const std::string& user_name, const std::string& password)
+    : user_name_(user_name), password_hash_(hashSHA512(password)), map_of_random_questions_and_answers_{}
+{
+}
+
+bool Account::isPasswordProper(const std::string& login, const std::string& password)
+{
+    std::ifstream file("users/users.json");
+    nlohmann::json jsonFile;
+    file >> jsonFile;
+
+    std::string hashedPassword = hashSHA512(password);
+
+    for (auto& element : jsonFile)
+    {
+        if (element[login] == login && element[password] == hashedPassword)
+        {
+            return true;
+        }
+    }
+
+    return false;
+}
+
+bool Account::areAnswersCorrect(const std::string& answer1, const std::string& answer2, const std::string& answer3)
```

Comment:
How is it supposed to be used? Will we see all 3 questions and we need to provide all answers at once?
It should be better to have a function that check 1 question at once.